src/components/storeWrapper.js

```diff
@@ -0,0 +1,16 @@
+export default function wrapStore(providedStore, ngReduxStore) {
+  const unsubscribe = providedStore
+    .subscribe(() => {
+      let newState = providedStore.getState();
+
+      ngReduxStore.dispatch(newState);
+    })
+  ;
+
+  return Object.assign({},
```

Comment:
Is the intention to not provide `.connect()` functionality when using a provided store? I fiddled with it and added it back in, the only caveat is that connected actions dispatch to the wrong store, so you always need to use the `$ngRedux.dispatch`.